tests/php/src/Theming/ThemeSupportTests.php

```diff
@@ -0,0 +1,252 @@
+<?php
+/**
+ * Tests for ThemeSupport
+ *
+ * @package Automattic\WooCommerce\Tests\ThemeSupport
+ */
+
+use Automattic\WooCommerce\Theming\ThemeSupport;
```

Comment:
Should be `Automattic\WooCommerce\Tests\Theming\ThemeSupport`!

Reply:
That's the full name of the class being tested, not the namespace of the test itself. 🙂

Reply:
I must have been tired last night, sorry!